### PR TITLE
Update default user-agent (#2785)

### PR DESCRIPTION
**Pull request by asieira**
* Updated default user agent so that it includes only the requests
version;
* Added PyCharm / IDEA project directory to `.gitignore`.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -15,6 +15,7 @@ requests.egg-info/
 *.swp
 *.egg
 env/
+.idea/
```
Comment (sigmavirus24) [inline .gitignore]: Please remove this. Pull requests should be singly purposed and not include unrelated changes.

**Comment by Lukasa** [comment]
If we can clean this up relatively swiftly it'll make it into 2.8.0: otherwise, it'll fall out until 2.8.1.

File: .gitignore (hunk 1 above)
Comment (asieira) [inline .gitignore]: It seemed like the right thing to do since there was no CONTRIBUTING document explaining what the rules were and this seemed like the most straightforward way to avoid getting the project files into the code both for me and future contributors who happen to use PyCharm / IDEA. 

You recommend I open a separate issue just for this one-line change to .gitignore, then? If that's the right thing to do, I'll do it.

**Comment by sigmavirus24** [comment]
I wouldn't be comfortable including this in 2.8.1. 2.9.0 definitely, but not 2.8.1

**Comment by Lukasa** [comment]
@asieira No, I think I recommend adding it to your local `.git/info/exclude` file. Text-editor specific config really shouldn't go into the project repository, the world just gets really complex really fast that way.

File: .gitignore (hunk 1 above)
Comment (sigmavirus24) [inline .gitignore]: No, I'm recommending you remove this line and use a [Global gitignore file](https://help.github.com/articles/ignoring-files/#create-a-global-gitignore).

**Comment by asieira** [comment]
Ok, will do that.

**Comment by Lukasa** [comment]
LGTM. @sigmavirus24?

**Comment by sigmavirus24** [comment]
@Lukasa will you handle the release notes for this or should I do it later tonight?

**Comment by Lukasa** [comment]
@sigmavirus24 I'll do it.

@asieira Thanks so much for this change! :sparkles: :cake: :sparkles:

**Comment by asieira** [comment]
:bow: my pleasure, thank you all again for the awesome package. :)
